crypto/src/signature.rs

```diff
@@ -52,6 +59,47 @@ impl Signature {
     pub fn inner_curve_scalar(&self) -> &InnerCurveScalar {
         &self.0 .1
     }
+
+    pub fn to_base58(&self) -> EncodeBuilder<'static, Vec<u8>> {
```

Comment:
This should be in the implementation of `Base58Encodeable`. You can add a custom implementation of to_base58 from that trait no need to use the default one

Reply:
Fixed.